indico/htdocs/sass/partials/_forms.scss

```diff
@@ -197,7 +199,9 @@ $i-form-button-line-height: 1.8;
     @include apply-to-text-inputs('%form-field-input');
     textarea {
         @extend %form-field-input;
-        height: auto;
+        &:not(.wmd-input) {
+            height: auto;
+        }
```

Comment:
Probably not needed anymore - it was breaking the auto size change but since that's gone maybe this is obsolete now